ironfish/src/network/peers/peer.ts

```diff
@@ -190,6 +192,16 @@ export class Peer {
 
   loggedMessages: Array<LoggedMessage> = []
 
+  /**
+   * Blocks that have been sent or received from this peer. Value is set to true if the block was received
+   * from the peer, and false if the block was sent to the peer.
+   */
+  readonly knownBlockHashes: LRU<Buffer, boolean> = new LRU<Buffer, boolean>(
+    1024,
```

Comment:
since we don't provide getSize() this is how many units. So this should be about `1024 hashes * 32 bytes * 50 peers` which will be about 1.6 MiB.